plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -216,3 +299,28 @@ func createESClient() (*elastic.Client, error) {
 		elastic.SetURL(queryURL),
 		elastic.SetSniff(false))
 }
+
+func getVersion(client *elastic.Client) (uint, error) {
+	pingResult, _, err := client.Ping(queryURL).Do(context.Background())
+	if err != nil {
+		return 0, err
+	}
+	esVersion, err := strconv.Atoi(string(pingResult.Version.Number[0]))
+	if err != nil {
+		return 0, err
+	}
+	return uint(esVersion), nil
+}
+
+func createILMPolicy(client *elastic.Client, policyName string) error {
+	_, err := client.XPackIlmPutLifecycle().Policy(policyName).BodyString("{\"policy\": {\"phases\": {\"hot\": {\"min_age\": \"0ms\",\"actions\": {\"rollover\": {\"max_age\": \"1d\"},\"set_priority\": {\"priority\": 100}}}}}}").Do(context.Background())
```

Comment:
Use backticks to avoid escaping quotes: `` `"foo"` ``

Reply:
Done